libs/MeshKernel/src/Contacts.cpp

```diff
@@ -53,7 +54,29 @@ void Contacts::ComputeSingleContacts(const std::vector<bool>& oneDNodeMask,
         {
             m_mesh1dIndices.emplace_back(n);
```

Comment:
push_back?

Reply:
should not make a difference for named variables